lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/ipv-core-main-journey.yaml

```diff
@@ -469,6 +469,38 @@ CRI_HMRC_KBV_J6:
     next:
       targetState: PYI_NO_MATCH
 
+# No photo id M2B journey (J7)
+CRI_BANK_ACCOUNT_J7:
+  response:
+    type: cri
+    criId: bav
+    context: bank_account
+  parent: CRI_STATE
+  events:
+    next:
+      targetState: CRI_NINO_WITH_SCOPE_J7
+
+CRI_NINO_WITH_SCOPE_J7:
+  response:
+    type: cri
+    criId: nino
+    scope: identityCheck
```

Comment:
Whats `scope` used for?

Reply:
So we can [reuse NINO CRI](https://govukverify.atlassian.net/wiki/spaces/Architecture/pages/3754755081/NINO+CRI+Reuse#Option-3%3A-Reuse-the-NINO-CRI%2C-adding-a-scope-to-determine-when-to-generate-an-identity-VC) and provide us with additional fields for the no photo id journey